.github/workflows/Documentation.yml

```
@@ -14,11 +14,12 @@ jobs:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@latest
         with:
-          version: '1'
+          version: '1.10'
```

```suggestion
          version: '1'
```

we should keep it on latest